packages/runtimes/js/src/JSRuntime.js

```diff
@@ -134,19 +134,19 @@ export default new Runtime({
 
       let bundleGroup = resolved.value;
       let bundlesInGroup = bundleGraph.getBundlesInBundleGroup(bundleGroup);
+      let mainBundle = bundlesInGroup[bundlesInGroup.length - 1];
```

Comment:
can this look for the bundle whose main entry asset id matches the bundle group's `entryAssetId`?

Reply:
Just to make sure: could there be a situation where there is no such bundle?

Reply:
Not that I know of — this makes sense as an invariant.